downloaded/lisk_bridge.sh

```diff
@@ -45,9 +45,11 @@ fi
 if [[ $SHOW_USAGE ]]; then
 	echo "Usage: $0 <-h <BLOCKHEIGHT>> [-s <DIRECTORY>] [-n <NETWORK>] "
 	echo '-h <BLOCKHEIGHT> -- specify blockheight at which bridging will be initiated'
+	echo '-f <TARBALL> -- Specify local tarball for destination release'
```

Comment:
I don't think "destination release" is clear. Maybe "specify path to local tarball containing the target release"?